docs/docs/gatsby-on-linux.md

```diff
@@ -2,12 +2,128 @@
 title: Gatsby on Linux
 ---
 
-> This is a TODO. Help our community expand it.
+# Linux
 
-> Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your pull request gets accepted.
+This guide assumes you already have a native installation of Linux on your machine. The following steps walk through how to install `node` and associated depdendencies.
+
+### Ubuntu, Debian and other `apt` based distros
+
+Begin by updating and upgrading.
+
+```shell
+sudo apt update
+sudo apt -y upgrade
+```
+
+> Only use `-y` if you want to upgrade to the latest version of the software.
+
+Install curl which allows you to transfer data and download additional dependencies.
+
+```shell
+sudo apt install curl
+```
+
+Once `curl` is installed, you can use it to install `nvm`, which will manage `node` and all its associated versions.
+
+```shell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
+```
+
+> Note that this is the current stable release of nvm. Full installation instructions and troubleshooting can be found at the [nvm GitHub page](https://github.com/nvm-sh/nvm)
+
+When `nvm` is installed, it does not default to a particular `node` version. You'll need to install the version you want and give `nvm` instructions to use it. This example uses `10.17.0`, but other version numbers can be used instead.
```

Comment:
IIRC with `nvm` you can just ask it to install the major version, and it will automatically pick up the latest release, i.e. `nvm install 10`

Reply:
This way we don't have to update the materials when a new minor/patch comes out